pkg/services/loginattempt/loginattemptimpl/store_test.go

```diff
@@ -0,0 +1,139 @@
+package loginattemptimpl
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/services/loginattempt"
+	"github.com/grafana/grafana/pkg/services/sqlstore"
+	"github.com/stretchr/testify/require"
+)
+
+func mockTime(mock time.Time) time.Time {
+	getTimeNow = func() time.Time { return mock }
+	return mock
+}
```

Comment:
We can turn this dog override to a cat override (…no?) by having it cover its tracks. WDYT?

```suggestion
func mockTime(t testing.TB, mock time.Time) time.Time {
	oldTimeNow = getTimeNow
	t.Cleanup(func() {
		getTimeNow = oldTimeNow
	})
	getTimeNow = func() time.Time { return mock }
	return mock
}
```

Reply:
I actually decided to implement `now()` as a struct field and not a global mock. Easier to fake, no need to clean up later.

Reply:
Ready for parallelism 💯